media/system/js/share-uncompressed.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @package         Joomla.JavaScript
+ * @copyright       Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+/**
+ * Calls the sending process of the config class
+ */
+jQuery(document).ready(function ($)
+{
+    $('#sharebutton').click(function ()
+    {
+        var share_draft = {
+            articleId  : $('jform_id').val(),
+            title: $('jform_title').val(),
```

Comment:
As you are using jQuery you forgot to use the Hash (#) here, e.g. $('#jform_id')

Btw.: When you want to use $, you need to make jQuery is used, either with $ = jQuery or surrounding your whole document ready function with:

(function($){
  $(document).ready...
 ...
 // End of your code
})(jQuery);

I would suggest using the last one here.